docs/mine/lotus/dagstore.md

```diff
@@ -0,0 +1,191 @@
+
+```
+2021-08-09T22:06:35.701+0300    INFO    dagstore.migrator       dagstore/wrapper.go:286 registering shards for all active deals in sealing subsystem    {"count": 453}
+2021-08-09T22:06:35.701+0300    WARN    dagstore.migrator       dagstore/wrapper.go:335 deal has nil piece CID; skipping        {"deal_id": 0}
+2021-08-09T22:06:35.701+0300    INFO    dagstore.migrator       dagstore/wrapper.go:348 registering deal in dagstore with lazy init     {"deal_id": 2208881, "piece_cid": "baga6ea4seaqhnvxy55e
+nveknyqhkkh7mltcrrcx35yvuxdmcbfouaafkvp6niay"}
+2021-08-09T22:06:35.702+0300    INFO    dagstore.migrator       dagstore/wrapper.go:318 async shard registration completed successfully {"shard_key": "baga6ea4seaqhnvxy55enveknyqhkkh7mltcrrcx
+35yvuxdmcbfouaafkvp6niay"}
+[...]
+2021-08-09T22:06:35.709+0300    INFO    dagstore.migrator       dagstore/wrapper.go:361 finished registering all shards {"total": 44}
+[...]
+2021-08-09T22:06:35.826+0300    INFO    dagstore.migrator       dagstore/wrapper.go:365 confirmed registration of all shards
+2021-08-09T22:06:35.826+0300    INFO    dagstore.migrator       dagstore/wrapper.go:372 successfully marked migration as complete
+2021-08-09T22:06:35.826+0300    INFO    dagstore.migrator       dagstore/wrapper.go:375 dagstore migration complete
+```
+
+## Forcing bulk initialization
+
+Forcing bulk initialization will become important in the near future, when miners begin publishing indices to the network to advertise content they have, and new retrieval features become available (e.g. automatic shard routing).
+
+You should start now if possible, as this process is better carried out gradually and over a longer timeframe, if you have many storage deals.
+
+You can force bulk initialization using the `lotus-miner dagstore initialize-all` command. This command will force initialization of every shard that is still in `ShardStateNew` state. To control the operation:
+- You must set a concurrency level through the `--concurrency=N` flag.
+  - A value of `0` will disable throttling and all shards will be initialized at once. ⚠️ Use with caution!
+- By default, only unsealed pieces will be indexed to avoid forcing unsealing jobs. To index also sealed pieces, use the `--include-sealed` flag.
+
+Initialization places IO workload on your storage system. You can stop/start this command at your wish/convenience as proving deadlines approach and elapse, to avoid IOPS starvation, or competition with window PoSt.
+
+To stop this command, press Control-C. Shards being initialized at that time will continue in the background, but no more initializations will be performed. The next time you run the command, it will resume from where it left off.
+
+::: tip
+In our test environments, we found the migration to proceed at a rate of 400-500 shards/deals per second.
```

Comment:
Elaborate more on what our test environment is like - specs so folks can have a reference to set expectation

Reply:
@nonsense could you add the specs for sofiaminer?